backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/VdsEventListener.java

```diff
@@ -626,4 +630,9 @@ private RunVmParams buildRunVmParameters(Guid vmId) {
     public ActionReturnValue saveExternalData(SaveVmExternalDataParameters parameters) {
         return backend.runInternalAction(ActionType.SaveVmExternalData, parameters);
     }
+
+    public void clearVmNuma(Guid vmId) {
+        backend.runInternalAction(ActionType.RemoveVmNumaNodes,
+                new VmNumaNodeOperationParameters(vmId, vmNumaNodeDao.getAllVmNumaNodeByVmId(vmId)));
```

Comment:
it would be better to avoid querying the nodes - if we can introduce a remove procedure that removes it for several vms (see comment below) that would be even better